.travis.yml

```diff
@@ -4,8 +4,7 @@ cache: pip
 matrix:
   include:
   - env: TOXENV=isort-check
-  - python: 3.6
-    env: TOXENV=lint
+  - env: TOXENV=lint
```

Comment:
I would feel more comfortable if __isort-check__ and __lint__ were run on Python 3.7 instead of 2.7.  This was done in #731 and the tests pass.  There are several keywords and built-ins like __async__, __basename__, raw_input__, __unicode__, __xrange__ that we want the lint step to ensure are handled properly.  Python 3 is almost always more strict in this regard than Python 3.

Reply:
In tox.in, `lint` is defined with `basepython = python2.7`. So it doesn't matter what the default system Python is, tox will always use `python2.7`. Therefore, I don't think changing the python version here adds any value.

https://github.com/timothycrosley/isort/blob/fcd80d46e66179f9464c2df745ffed07622d9ee3/tox.ini#L14-L15

Here is the tox documentation on the configuration option:

https://tox.readthedocs.io/en/latest/config.html#confval-basepython=NAME-OR-PATH

Removing the `basepython` causes the tox target to fail for me. So I don't think the lint target is compatible with Python 3 yet.